[WIP] [DOCS] Add Nebari installation guides and getting-started as first directions (#45)
nebari-dev/nebari: docs/docs/how-tos/domain-registry.md

---
id: domain-registry
title: How to set Nebari's DNS Domain Registry
---

# Nebari domain registry setup

This domain will be where your application will be exposed.

> Currently, Nebari only supports CloudFlare for automatic DNS registration. If an alternate DNS provider is desired, change the `--dns-provider` flag from **cloudflare** to **none** on the Nebari deploy command. The deployment then will be paused when it asks for an IP address (or CNAME, if using AWS) and prompt to register the desired URL.

Reviewer: ```suggestion
Currently, Nebari only supports CloudFlare for automatic DNS registration. If an alternate DNS provider is desired, change the `--dns-provider` flag from **cloudflare** to **none** on the Nebari deploy command. The deployment process will then pause when it asks for an IP address (or CNAME, if using AWS) and prompt you to provide the desired URL.
```

I *think* this is an improvement... without having actually done a deployment, I can't be sure if I understand what's happening here, but "register" seems like the wrong word for a user pasting an IP/CNAME into the CLI.

Author: oh! I just populated these files with part of what's currently present in our docs -- That whole section needs to be written!

Author: for ref. https://docs.qhub.dev/en/stable/source/installation/setup.html?highlight=cloudflare#domain-registry

Reviewer: No, I think it's ok - it looks like it just got split to a new paragraph.

> Setting a DNS record heavily depends on the provider thus it’s not possible to have detailed docs on how to create a record on your provider. Googling setting **A/CNAME** record on your requested cloud provider should yield good results on doing it for your specific provider.

Reviewer: ```suggestion
Setting a DNS record heavily depends on the provider, and thus it’s not possible to provide detailed docs here on how to create a record on all of the providers out there. Searching "setting A DNS record" or "setting CNAME DNS record" on your specific cloud provider should yield good results.
```

I might have broken your intent with the change around search for A/CNAME record results...sorry if so!

I think the generic 'searching' is probably better than 'googling', even if the term is basically genericized at this point.
